tasks/components.py

```diff
@@ -120,6 +124,7 @@ def get_components_and_bundles():
 
         component_directory = pathlib.Path(component_file)
         for component_entry in component_directory.iterdir():
+
             # If we encounter a file at the first level it could be a bundle
             if component_entry.is_file():
                 if component_entry.name == "bundle.go":
```

Comment:
why here do not use: `if component_entry.is_file() && component_entry.name == "bundle.go":`
So later you can remove continue instruction and add an `elif component_entry.is_file():`
It would avoid to go through a lot of `if` instruction no ?

Reply:
Combing the conditions with `and` is a good idea, will do. Using `continue` instead of `elif` allows the code to remove a layer of indentation, making it a bit easier to read.